.github/workflows/check-contrib.yml

```diff
@@ -0,0 +1,83 @@
+name: check-contrib
+
+on: [push, pull_request]  # would you need to define PRs containing .eb files?
```

Comment:
Since the workflow is triggered on all push and pull request events, should I add a conditional to only run the check-contrib step if .eb files are modified?

e.g. if: contains(github.event.pull_request.changed_files, '.eb')